src/components/application_manager/src/request_controller.cc

```diff
@@ -266,18 +271,20 @@ void RequestController::terminateWaitingForExecutionAppRequests(
     const uint32_t& app_id) {
   SDL_AUTO_TRACE();
   SDL_DEBUG("app_id: " << app_id << "Waiting for execution"
```

Comment:
@VProdanov It could be necessary - it shows how many requests are waiting execution before and after erasing from list